plugin/jvm/src/main/java/com/twosigma/beaker/jvm/object/SimpleEvaluationObject.java

```diff
@@ -467,6 +469,29 @@ private synchronized void doAppendError(String s) {
     }
   }
 
+  private synchronized void doFlush() {
```

Comment:
doFlush -> it does something and what it does is a flush.
this mimics other doXXX functions in the same file